components/merino/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod merino;
```

Comment:
Suggestion: Merino has many endpoints (suggest, curated_recommendation, manifest, etc.), how about naming the mods accordingly? E.g. `curation` for now, `suggest` and `manifest` can be added in the future.